styles/CSS/base.css

```diff
@@ -69,8 +79,22 @@ header h2 {
   width: 100%;
 }
 
+#extra {
+  margin-top: 10%;
+  border: dotted .5em orange;
+}
+
 #block {
   display: inline-block;
+  padding: 1% 2.5% 0 2.5%;
+}
+
+#articles{
+  margin-top: 25%;
+  border: dotted .5em grey;
+
+footer {
+  padding-top: 0.5%;
 }
 
 /*LINK STYLES
```

Comment:
It's so weird that your text-decoration property isn't removing the underline and color from your nav bar links. I know that when I have this problem, it's usually a matter of me now being specific enough in trying to capture the element.